packages/@okta/vuepress-site/docs/change-log/2019-04-1/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Okta API Products Change Log
+---
+
+## 2019.04.1
```

Comment:
When I try to build the web site from this branch locally on my machine, I don't see the new change log entry. I'm not sure what's going on. @bretterer, do you have time to take a look? This is what I see:
<img width="829" alt="Screen Shot 2019-04-19 at 10 43 40 AM" src="https://user-images.githubusercontent.com/43331081/56429308-72106300-6290-11e9-9222-9cbae4cb331b.png">

Reply:
Looks fine to me.

<img width="1337" alt="Screen Shot 2019-04-19 at 11 01 13 AM" src="https://user-images.githubusercontent.com/1906920/56429910-881f2300-6292-11e9-9982-14722bb6108d.png">

Please make sure you are not cached and have a current version of this branch

Reply:
Thanks for checking

Reply:
Seems I had to do `yarn install`, now seeing the page correctly.